elastic/kibana
# [SecuritySolution] Add auditing to Asset Criticality and Risk Engine

## Summary

Add auditing to Asset Criticality and Risk Engine as specified in the EPIC ticket.


All telemetry is sent from data clients and route files. To do that, I had to update some extra files to make sure the routes and data clients had access to the audit logger. 



### How to test
1. Deploy branch to cloud
2. Send cloud deployment logs to another deployment
3. Enable `xpack.security.audit.enabled: true` for kibana
4. Use the risk engine on the deployment
5. On the logs deployment, verify if the audit logs are arriving
   * Open explore
    * Set the data view `elastic-cloud-logs-*`
    * Search for the actions `event.action: action_name`
